client/app/intake/pages/formGenerator.jsx

```diff
@@ -59,113 +55,179 @@ const formFieldMapping = (props) => {
   };
   const renderVaGovValue = () => {
     // eslint-disable-next-line no-undefined
-    if (isAppeal && (props.filedByVaGov === null || props.filedByVaGov === undefined)) {
+    if ( isAppeal && (props.filedByVaGov === null || props.filedByVaGov === undefined)) 
+    {
       return 'false';
     }
 
     return renderBooleanValue('filedByVaGov');
   };
 
-  return ({
+  const hearingTypeDropdown = (
+    <SearchableDropdown
+      label="Please Select Hearing Type"
+      strongLabel
+      name="original-hearing-request-type"
+      onChange={({ value }) => props.setOriginalHearingRequestType(value)}
+      options={hearingTypeOptions}
```

Comment:
```suggestion
      options={hearingTypeOptions}
      optional
```
@minhazur9 according to the [designs](https://bahdigital.invisionapp.com/share/A2IAIBPS6V9#/screens/296070635) we want to show the user that the selection is _Optional_.  I think we can achieve this by adding the `optional` attribute to the Searchable dropdown props

Reply:
Would it be a good idea to just click on the "commit suggestion button" here if I know it works or is it better to commit locally and push?

Reply:
Yep I think that's fine.  You might want to double check your jest & feature tests to make sure it won't cause anything to fail there but I think you should be ok.